third_party/NordicSemiconductor/README.md

```diff
@@ -10,3 +10,11 @@ Directory consists of following folders:
  - /nrfx         - Standalone drivers for peripherals present in Nordic SoCs (https://github.com/NordicSemiconductor/nrfx)
  - /segger_rtt   - Library for the RTT communication
  - /softdevice   - SoftDevice s140 headers
+
+ The following changes comparing to the nRF5 SDK 15.2 have been incorporated:
+ - move definitions of CDC_ACM_COMM_INTERFACE and CDC_ACM_DATA_INTERFACE from usb-cdc-uart.c to platform-config.h,
```

Comment:
nit: let's remove first three bullet points, they are more related to OpenThread platform.